airflow/api_connexion/openapi/v1.yaml

```diff
@@ -3581,6 +3619,31 @@ components:
           description:
             The value can be repeated to retrieve multiple matching values (OR condition).
 
+    RunTaskInstanceForm:
+      type: object
+      required:
+        - ignore_all_deps
+        - ignore_task_deps
+        - ignore_ti_state
+        - map_index
```

Comment:
Should we have this on the fields themselves? Did you check how it appears on documentation? Is map_index required or it should have a default

Reply:
I made all fields required. I will leave it to @bbovenzi if this needs any change. API docs render as attached.
![task_instance_run_redoc](https://user-images.githubusercontent.com/3972343/169636252-642f2817-a58d-46ab-91c7-83543c7c3a0f.png)
.
![task_instance_run_swagger](https://user-images.githubusercontent.com/3972343/169636249-e257797e-96f3-4ef8-b5a2-787addac97e9.png)

Reply:
Actually, I'd say everything in the request body should be optional since we'd default to false, or all map_indexes.
Path parameters are required though

Reply:
I had made defaults as false for boolean and -1 for map_indexes (unmapped by default)

![run-task-instance-updated](https://user-images.githubusercontent.com/3972343/174785441-e1b1bc46-8bee-48a4-85e4-04bf1d853daf.png)